analyzer/src/test/kotlin/YamlFilePackageCurationProviderTest.kt

```diff
@@ -52,5 +53,29 @@ class YamlFilePackageCurationProviderTest : StringSpec() {
                 it.id.matches(identifier) shouldBe false
             }
         }
+
+        "Provider returns only matching curations for a version range" {
+            val provider = YamlFilePackageCurationProvider(curationsFile)
+
+            val idMinVersion = Identifier("npm", "", "ramda", "0.21.0")
+            val idMaxVersion = Identifier("npm", "", "ramda", "0.25.0")
+            val idOutVersion = Identifier("npm", "", "ramda", "0.26.0")
+
+            val curationsMinVersion = provider.getCurationsFor(idMinVersion)
```

Comment:
All code / testing for curationsMaxVersion is redundant with the code / testing for curationsMinVersion.
It can be removed.

Reply:
I don't believe it's redundant as I wanted to verify the full version range is covered (and not only any single border value of the interval).

Reply:
Should we also test a version between min and max, and other Ivy matchers like `0.21.+` or `(0.21.0, 0.25.0)`?

Reply:
As for @fviernau's taste already the existing test were too much, I'd propose to keep it as-is 😉